flexget/plugins/output/symlink.py

```diff
@@ -50,7 +51,12 @@ def on_task_output(self, task, config):
                 continue
             lnkfrom = entry['location']
             name = os.path.basename(lnkfrom)
-            lnkto = os.path.join(config['to'], name)
+            try:
+                lnkto = render_from_entry(os.path.join(config['to'], name), entry)
+            except RenderError as error:
+                log.error('Could not render path: %s', to)
```

Comment:
What is `to` in this context? Doesn't look like it's initialized anywhere.

Reply:
Yes, it should be `lnkto`.  Will fix it in few hours when I get back home.

Nice catch.

Reply:
Fixed